packages/docusaurus-1.x/lib/core/Head.js

```diff
@@ -172,6 +172,35 @@ class Head extends React.Component {
             }}
           />
         )}
+        <script src="https://unpkg.com/smooth-scroll@16.0.3/dist/smooth-scroll.polyfills.min.js" />
+        <script src="/js/scrollSpy.js" />
+        <script
```

Comment:
Why don't put the contents of this script into `scrollSpy.js`? We'd wanna avoid using strings where possible as they're harder to statically analyze.

(This is assuming we want to keep this scroll on click feature which I'm not in favor of)